addons/calendar/data/mail_template_data.xml

```diff
@@ -92,8 +92,8 @@
                 <t t-if="recurrent">
                     <li>When: <t t-out="object.recurrence_id.name or ''">Every 1 Weeks, for 3 events</t></li>
                 </t>
-                <t t-if="not object.event_id.allday and object.event_id.duration">
-                    <li>Duration: <t t-out="('%dH%02d' % (object.event_id.duration,round(object.event_id.duration*60)%60)) or ''">0H30</t></li>
+                <t t-if="object.event_id.duration">
+                    <li>Duration: <t t-out="object.event_id.duration" t-options="{'widget': 'duration', 'unit': 'hour', 'round': 'minute'}"/></li>
```

Comment:
This is a strange change. I end up with emails telling my meeting has a duration of 1 hour, but I can't change hours, and on the form view anyway only the day is displayed ... if you go full day, why displaying duration ?